app/src/main/java/com/karumi/ui/domain/usecase/GetSuperHeroes.kt

```diff
@@ -0,0 +1,16 @@
+package com.karumi.ui.domain.usecase
+
+import com.karumi.ui.domain.Result
+import com.karumi.ui.domain.model.SuperHero
+
+class GetSuperHeroes {
```

Comment:
As this language is less verbose you can create a file named ``usecases.kt`` and move the two use cases we will have to this file. You can even name it ``domain`` and move your abstract data types and use cases there.

Reply:
Not for me, one of the good part of the Use cases takes a look and check all operations that we can do over our domain, if we put all together in one file this file can grow to fast and is not a benefict for read it.

Reply:
ok